src/ActiveField.php

```diff
@@ -431,6 +431,8 @@ public function listBox($items, $options = [])
         if ($this->form->layout === ActiveForm::LAYOUT_INLINE) {
             Html::removeCssClass($this->labelOptions, 'visually-hidden');
         }
+        Html::removeCssClass($options, 'form-control');
```

Comment:
This seems unnecessary since it will only remove this from `$options` passed here which is working against the developer's will.

Reply:
@bizley you right